### PR TITLE
Update dummy atomic decorator 

### PR DESCRIPTION
**Pull request by bocribbz**
When used as `@atomic` it doesnt apply the patch correctly.

It should still work as `@atomic()` too.

### CODE REVIEW

File: nosedjangotests/polls/models.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ class Choice(models.Model):
     choice = models.CharField(max_length=200)
     votes = models.IntegerField(default=0)
 
-    @atomic()
+    @atomic
```
Comment (caffodian) [inline nosedjangotests/polls/models.py]: does this end up breaking the other form?  is it intended?

**Comment by caffodian** [comment]
seems like a reasonable/simple test :) :+1:

File: nosedjangotests/polls/models.py (hunk 1 above)
Comment (bocribbz) [inline nosedjangotests/polls/models.py]: This was my attempt to break some test, but it didnt break anything. 
The change in this PR is suppose to support @atomic replaced by the dummy context manager

**Comment by winhamwr** [comment]
It would have been nice to get a failing test created before fixing this. Would that have been particularly difficult because of transaction things?

**Comment by bocribbz** [comment]
We were unable to make a test fail even before, when we tried to fix `@atomic()`. 
Im not really sure what the entire context is to reproduce this issue.

**Pull request merged**